paymentsheet/src/test/java/com/stripe/android/paymentsheet/forms/FormViewModelTest.kt

```diff
@@ -59,6 +59,21 @@ internal class FormViewModelTest {
 
     private val showCheckboxFlow = MutableStateFlow(false)
 
+    @Test
+    fun `Verify completeFormValues is not null when no elements exist`() = runTest {
+        val args = COMPOSE_FRAGMENT_ARGS.copy(
+            paymentMethodCode = PaymentMethod.Type.Card.code
+        )
+        val formViewModel = createViewModel(
+            args,
```

Comment:
could you add parameter names to this method call? Otherwise it's hard to know what the emptyList is for